app/models/licence_transaction.rb

```diff
@@ -21,9 +22,17 @@ def self.title
     "Licence"
   end
 
+  def schema_organisations
+    return [] if @creator_organisation.blank?
+
+    [@creator_organisation]
+  end
+
   def primary_publishing_organisation
-    # Set to GDS for testing
-    "af07d5a5-df63-4ddc-9383-6a666845ebe9"
+    # Return GDS organisation ID if not supplied
+    return "af07d5a5-df63-4ddc-9383-6a666845ebe9" if @creator_organisation.blank?
```

Comment:
If a creating organisation doesn't exists then I think it probably makes sense to set the publishing org to BEIS (rather than us). GDS was just used for testing and should probably be swapped now?

Reply:
See comment above